pkg/openwsn/Makefile

```
@@ -0,0 +1,62 @@
+PKG_NAME=openwsn-fw
+PKG_URL=https://github.com/openwsn-berkeley/openwsn-fw.git
+PKG_VERSION=cbcf622bd9369fcfc8455a5fb9349de2ed3c3a46
+PKG_LICENSE=BSD-3-Clause
+
+include $(RIOTBASE)/pkg/pkg.mk
+
+# openwsn_% RIOT Modules or PSEUDOMODULES that don't have custom rules
+IGNORE_MODULES  := openwsn_leds \
+                   openwsn_debugpins \
+                   openwsn_radio \
+                   openwsn_serial \
+                   openwsn_sctimer% \
+                   openwsn_cryptoengine \
+                   #
+
+OPENWSN_MODULES := $(filter-out $(IGNORE_MODULES),$(filter openwsn_%,$(USEMODULE)))
+
+.PHONY: openwsn_%
+
+all: $(OPENWSN_MODULES)
+	"$(MAKE)" -C $(PKG_BUILDDIR) -f $(CURDIR)/Makefile.$(PKG_NAME)
```

#14289 was merged so this Makefile must be adapted (please rebase first): all occurrences of `PKG_BUILDDIR` must be replaced with `PKG_SOURCE_DIR`:
```suggestion
	"$(MAKE)" -C $(PKG_SOURCE_DIR) -f $(CURDIR)/Makefile.$(PKG_NAME)
```

---

By the way, this line could be removed, as well as the `Makefile.openwsn-fw` and `openwsn-fw` be declared as a pseudo-module in Makefile.include.

---

:(